tests/manual-test-cases/Group19-VIC-Product/19-10-VIC-Vulnerability.md

```diff
@@ -0,0 +1,35 @@
+Test 19-10 VIC Vulnerability scanning
+=======
+
+# Purpose:
+To verify that the vulnerability scanning feature of VIC product works as expected
+
+# References:
+[1- VIC vulnerability scan](TBD - waiting on official docs to link)
+
+# Environment:
+This test requires that a vSphere server is running and available
+
+# Test Steps:
+1. Download and install the VIC OVA from here:
+[OVA Releases](https://console.cloud.google.com/storage/browser/vic-product-ova-releases/?project=eminent-nation-87317&authuser=1)
+2. Register Harbor as a registry with Admiral
+3. Do a docker push to the harbor instance and populate the harbor instance with images that can be pulled
+4. Do a docker login/pull from the harbor instance
+5. Login to Admiral UI, Go to ‘Administration->Projects->Repositories tab’ locate the specified repository
+6. Click on the arrow to expand the embedded tag list of the specified repository and then click on the vertical ellipsis to open the popup menu.
+7. Click on 'Scan' to start the vulnerability analysis process
+8. Verify different scanning status "Not Scanned","Queued","Scanning" and "Complete"- status are indicated in the 'Vulnerability' column
+9. Verify different scanning status "Not Scanned","Queued","Scanning" and "Error" - status are indicated in the 'Vulnerability' column
+10. Verify that if error occurred, user can click on the 'View Log'
+11. Verify result once the scan has been completed - Red: High, Orange: Medium, Yellow: Low, Grey: Unknown and Green: No vulnerabilities
```

Comment:
It's not clear how to check that each of these statuses is present. Should there be a different image for each one or is there a procedure to trigger each value?